### DISCUSSION TITLE
Does Popover setContent variable look right under v5.3 documentation?

### DISCUSSION THREAD

**Discussion by Git-Harshit** [twbs/bootstrap#39280] [Polls]
I was seeking popover usage example with Bootstrap v5.3, when I noted 'popover' const variable assigned above with `.Popover.getOrCreateInstance()` but following it, setContent method was used with 'myPopover' variable which isn't shown anywhere above in that page (content found below [Methods table](https://getbootstrap.com/docs/5.3/components/popovers/#methods) and right above [Events heading](https://getbootstrap.com/docs/5.3/components/popovers/#events)) as shown below:
![image](https://github.com/twbs/bootstrap/assets/57865187/29cceef6-f6f3-4957-b3a8-7eebf17e3852)

**Comment by julien-deramond** [comment]
Good catch @Git-Harshit, it should be `popover.setContent` in the documentation. A PR has been created in order to fix the documentation: https://github.com/twbs/bootstrap/pull/39281.

**Comment by Git-Harshit** [comment]
Thanks for your confirmation @julien-deramond  as well as a quick PR to fix it, quick enough to get the immediate next number to this discussion as Pull Request ID.
Missed your vote above 🙃
